partiql-value/src/lib.rs

```diff
@@ -151,12 +300,65 @@ impl Ord for Value {
             (Value::Boolean(_), _) => Ordering::Less,
             (_, Value::Boolean(_)) => Ordering::Greater,
 
+            // TODO: `OrderedFloat`'s implementation of `Ord` slightly differs from what we want in
```

Comment:
Since there are a lot of cases here, it likely warrants a unit test, not just relying on the conformance tests that will eventually assert on it.
It's likely as 'simple' as creating a `Vec` of `Value` and then `.sort()`ing it and asserting on the output order.